activation/interface.go

```diff
@@ -78,6 +78,33 @@ type SmeshingProvider interface {
 	UpdatePoETServers(ctx context.Context, endpoints []string) error
 }
 
+// poetProvingServiceClient provides a gateway to a trust-less public proving service, which may serve many PoET
+// proving clients, and thus enormously reduce the cost-per-proof for PoET since each additional proof adds only
+// a small number of hash evaluations to the total cost.
+type poetProvingServiceClient interface {
```

Comment:
As you are already changing it. Why not just:
```suggestion
// poetClient provides a gateway to a trust-less public proving service, which may serve many PoET
// proving clients, and thus enormously reduce the cost-per-proof for PoET since each additional proof adds only
// a small number of hash evaluations to the total cost.
type poetClient interface {
```

By the way, the description in the comment is weird and not helpful in my opinion.